percy/providers/generic_provider.py

```diff
@@ -22,8 +25,9 @@ def supports(_remote_url):
     def screenshot(self, name, **kwargs):
         tiles = self._get_tiles(**kwargs)
         tag = self._get_tag(**kwargs)
+        ignore_regions = self._find_ignored_regions(**kwargs)
 
-        return self._post_screenshots(name, tag, tiles, self.get_debug_url())
+        return self._post_screenshots(name, tag, tiles, ignore_regions, self.get_debug_url())
```

Comment:
this is incorrect order of args, we should have a test that checks for this, clearly that test is missing

Reply:
it is not missing it's just that we have mocked response in test_post_screenshot so our test are running regardless.